EIPS/eip-rich-site-proposed-contract-metadata.md

```diff
@@ -0,0 +1,93 @@
+---
+eip: 5163
+title: Rich Site-Proposed Contract Metadata
+description: A protocol for dapps to suggest metadata to wallets related to relevant contracts.
+author: Dan Finlay (@danfinlay)
+discussions-to: https://ethereum-magicians.org/t/eip-rich-site-proposed-contract-metadata/9635
+status: Draft
+type: Standards Track
+category: Interface
```

Comment:
```suggestion
category: ERC
```
ERC is probably the better category for this.

Reply:
I agree with @Pandapip1 here.